src/main/java/com/revature/controllers/PersonController.java

```diff
@@ -52,32 +51,72 @@
 	@RequestMapping(method = RequestMethod.GET, value = "persons")
 	public ResponseEntity<Page<Person>> getPerson(Pageable pageable, @RequestParam(defaultValue="", required=false) String firstname,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "getPerson" is 29 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)